l10n/en/products/vpn/seo/desktop.ftl

```diff
@@ -0,0 +1,48 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
+
+
+# Shared
+vpn-shared-cta = See pricing & availability
+vpn-shared-day-guarantee = 30-day money-back guarantee
+vpn-shared-what-youll-get = What you’ll get with { -brand-name-mozilla-vpn }:
+
+vpn-shared-feature-item-protection = Protection for up to 7 devices
+vpn-shared-feature-item-access = Access to servers in 30+ countries
+vpn-shared-feature-item-encryption = Device-level encryption
+vpn-shared-feature-item-bandwidth = No bandwidth restrictions
+vpn-shared-feature-item-logs = No online activity logs now or ever
+
+# Desktop strings
+vpn-desktop-hero-headline = Secure your internet connection with { -brand-name-mozilla-vpn } for desktop
+vpn-desktop-hero-copy = Get fast, easy security on your desktop and laptop computer through { -brand-name-mozilla }, one of the most trusted names in technology.
+
+vpn-desktop-guarantee = 30-day money-back guarantee
+
+vpn-desktop-ability = { -brand-name-mozilla-vpn } gives you the ability to connect confidently and keep your information safe from prying eyes and data thieves when you’re online. { -brand-name-mozilla-vpn } uses advanced WireGuard® protocol to encrypt personal data on your laptop or desktop computer — for up to five devices. Stream shows, play games, shop, and go about your daily life online knowing that you’re secure.
```

Comment:
Includes the `WireGuard™` string.

Reply:
Wireguard should be added as a brand name to brands.ftl.

Reply:
Done! I moved all the shared terms into a `shared.ftl` for the platform pages.